### DISCUSSION TITLE
Most common use cases of VBET I forsee

### DISCUSSION THREAD

**Discussion by joewheaton** [Riverscapes/riverscapes-tools#11] [VBET] [answered]
@philipbaileynar, the value in VBET beyond panning and zooming and exploring will be:

1. Using VBET output as a starting point, and choosing a threshold and then "editing' that polygon based on other subjective and judement based manual evidence. 
2. Re-running VBET with my own LiDAR for my segment (credit card please)
3. Re-running VBET with my own manually edited reach segment network geometry  (credit card please initially)
4. Re-running or filtering VBET with my own selection of what part of network to use
5. Re-running VBET with my own loaded network (credit card please)
6. Re-running VBET with my own parameterization

My big questions are:
1. Which of above scare you?
2. Do you see away for any of above to be part of same Riverscapes VBET project, as different realizations (I see 1 as adding a new realization of an output; 2 - 6 as new realization of input)

**Comment by joewheaton** [comment]
Reminder... @philipbaileynar

**Comment by joewheaton** [comment, marked as answer]
@philipbaileynar, this was from April. You and I have discussed this a lot since then. Please consider how and if are thinking has shifted since then and propose the answer here that you think we are dealing with in:
- A) Year 1 BLM Deliverable of Commercial Grade VBET
- B) Year 2 BLM Upgrade to Commercial Grade VBET
- C) Riverscapes Studio - Desktop QGIS
- D) Riverscapes Studio - Web (Year 3 BLM)?
- E) RAVE consumption

My re-read is that of the use cases identified in April
- 1) → Maps to B and C and later D
- 2) → Maps to D?
- 3) → Maybe slips between cracks as custom request... Maybe Year 4 Riverscapes Studio - Web?
- 4) → Maps to D (i.e. FY 2023)
- 5) → Maybe slips between cracks as custom request... Maybe Year 4 Riverscapes Studio - Web?

I think we've added a some new use cases focused very much around the "My Riverscapes" idea of 1, but really user-centric.

@nick4rivers you should see this too as it tracks what you and @KellyMWhitehead have already done in RIPT.

**Reply by joewheaton** [reply]
@shelbysawyer you should be reading this and tracking.

**Reply by shelbysawyer** [reply]
> @philipbaileynar, this was from April. You and I have discussed this a lot since then. Please consider how and if our thinking has shifted since then and propose the answer here that you think we are dealing with in:
> 
> * A) Year 1 BLM Deliverable of Commercial Grade VBET
> * B) Year 2 BLM Upgrade to Commercial Grade VBET
> * C) Riverscapes Studio - Desktop QGIS
> * D) Riverscapes Studio - Web (Year 3 BLM)?
> * E) RAVE consumption
> 
> My re-read is that of the use cases identified in April
> 
> * i. → Maps to B and C and later D
> * ii. → Maps to D?
> * iii. → Maybe slips between cracks as custom request... Maybe Year 4 Riverscapes Studio - Web?
> * iv. → Maps to D (i.e. FY 2023)
> * v. → Maybe slips between cracks as custom request... Maybe Year 4 Riverscapes Studio - Web?
> 
> I think we've added a some new use cases focused very much around the "My Riverscapes" idea of 1, but really user-centric.

@joewheaton I'm not fully understanding what you're laying out here. What are the 'i-v' bullets referring to? The original list, numbered '1-6'?

**Reply by joewheaton** [reply]
This conversation has moved to UI Specs in #389 and #390
